### PR TITLE
[ADD] essentials: contacts 

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3368806&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@jero-odoo This PR is ready for your review when you have a moment. Thanks, 👍🏻Tim

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:6]:
```suggestion
the company does business with through Odoo. A contact is a repository of vital business
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:12]: add :menuselection: markup for Contacts app

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:16]:
```suggestion
Odoo allows for both :guilabel:`Individual` and :guilabel:`Company` contacts. Select either
:guilabel:`Individual` or :guilabel:`Company`, depending on the type of contact that is being added.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:26]:
```suggestion
will appear throughout the database. This *name*  field is mandatory.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:30]:
```suggestion
   selecting :guilabel:`Individual`, a new field, called :guilabel:`Company`, appears below the first
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:39]:
```suggestion
   If an :guilabel:`Individual` is chosen, then the *type of address* can be chosen. Options for this
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:59]:
```suggestion
- :guilabel:`Website`: enter the full website address, starting with `http` or `https`.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:92]:
```suggestion
Once an option is selected, enter the corresponding contact information that should be
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:101]:
```suggestion
Set :guilabel:`Job Position`, which appears if the :guilabel:`Contact` address type has
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:104]:
```suggestion
There is also a :guilabel:`Note` field down here, as well. To make one, click on the text field
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:118]:
```suggestion
typing the user's name and making the appropriate selection.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:131]:
```suggestion
Lastly, if the company purchases products from this contact, one of the database users can be
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:139]:
```suggestion
The :guilabel:`Accounting` tab appears when the *Accounting* application is installed. Here, a user
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:153]:
```suggestion
partner options, including :guilabel:`Partner Activation` and :guilabel:`Partner Review`
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:167]:
```suggestion
At the top of the :guilabel:`Contact` form are some additional options available as *smart buttons*.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:177]:
```suggestion
   If the corresponding applications are installed, their related smart buttons appear
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:174]:
```suggestion
   related to this customer appear from the *CRM* app.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:190]:
```suggestion
There is also a :guilabel:`Go to Website` smart button. If the contact is a partner, the user can visit their partner page on the Odoo-built website.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:195]:
```suggestion
Now, if a user decides that they no longer want to have this contact active, the record can
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:208]:
```suggestion
   this is done, the :guilabel:`Archived` banner is removed, and the contact is restored.
```

File: content/applications/essentials/contacts.rst
Comment (jero-odoo) [inline content/applications/essentials/contacts.rst:219]: the only thing wrong with the icon is that it's not spinning :)

**Review by jero-odoo — APPROVED** [review]
Hey @tiku-odoo this is all set, great work!  Approving now with suggestions.
And thank you for the easter egg :)

File: content/applications/essentials/contacts.rst
Comment (StraubCreative) [reply content/applications/essentials/contacts.rst:219]: 100%

**Comment by jero-odoo** [comment]
Hey @ksc-odoo, this is ready for final review (Tiku started this, I did the first peer review, and I am taking it over from here). Thanks!

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
hey, @jero-odoo -- just finished my Final Review. And, I totally understand that you picked this up from Tim after he left, and you were just filling in the blanks that he left. That said, great work! 👍 

However, there are some instances that could benefit from a double-check during your revisions, just to make sure the tabs are featuring the same content that is being mentioned in the doc, as I encountered a handful of differences during my run-through using a 15.0 runbot -- but I'm sure those portions will be quick to look at and fix (if needed) 

If you have any questions about any of my feedback, don't hesitate to reach out. Anyway, once you give those elements a check, and think everything is good to go, tag me again for another quick look. 👀 

Thanks! 🙏

**Comment by jero-odoo** [comment]
Thanks @ksc-odoo this is ready for another look. (I changed the Sales & Purchasing section to tabs, just because it was a bit long. Let me know if you think this works or not, since it is the only one that is formatted that way, I cannot tell if it makes things concise or just disrupts the flow, I would appreciate a second opinion.)
Thanks!

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:154]:
```suggestion
```
suggesting the deletion of an extra empty line

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:155]:
```suggestion
Under the :guilabel:`Fiscal Information` heading, select a :guilabel:`Fiscal Position` from the drop-down
```

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:164]:
```suggestion
Under the :guilabel:`Miscellaneous` heading, use the :guilabel:`LEI` field to enter a Legal Entity
```

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:191]:
```suggestion
At the top of the contact form, there are some additional options available, known as
```

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:198]:
```suggestion
   For example, there is an :guilabel:`Opportunities` smart button, where all the opportunities
```

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:199]:
```suggestion
   related to this customer from the *CRM* app are accessible.
```

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:203]:
```suggestion
   automatically on a contact form.
```

File: content/applications/essentials/contacts.rst
Comment (ksc-odoo) [inline content/applications/essentials/contacts.rst:226]:
```suggestion
With this contact successfully archived, as indicated by a banner at the top, they do not show up
```

**Review by ksc-odoo — APPROVED** [review]
Alrighty, @jero-odoo -- everything is looking good to me -- just have a handful of minor suggestions/comments that require your attention. but, once you make those adjustments, feel free to tag this for tech review. approving now! 👍

**Comment by jero-odoo** [comment]
Thanks KC!
@samueljlieber this is ready for tech review. Thanks!

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:117]: "Below that" doesnt make a lot of sense to me. It would be helpful to use more specific language, as well the remaining Fiscal Position instructions come after the tabs... lets keep these instructions together :)
```suggestion
The :guilabel:`Fiscal Position` can be set on the :guilabel:`Sales & Purchases` tab. Select a
:guilabel:`Fiscal Position` from the drop-down menu.
```

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:157]: I dont think using tabs here is very effective, here is my reasoning. [Tab headings cannot be translated](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#content-tabs), as well the information inside hidden tabs cannot be searched for (using `ctrl + f` in browser). I think all of this information is important to show up-front. So suggesting to use regular-ole' sections here :)
```suggestion
Sales section
~~~~~~~~~~~~~

Under the :guilabel:`Sales` heading, a specific :guilabel:`Salesperson` can be assigned to a
contact. To do that, click the :guilabel:`Salesperson` drop-down field, and select one. Create a new
:guilabel:`Salesperson` by typing the user's name, and making the appropriate selection.

Certain :guilabel:`Payment Terms`, or a certain :guilabel:`Pricelist`, can also be set, if needed.
Click the drop-down menu next to :guilabel:`Payment Terms`, and change it to one of the preselected
:guilabel:`Payment Terms`, or :guilabel:`Create` a new one. Select the :guilabel:`Pricelist`
drop-down menu to choose the appropriate :guilabel:`Pricelist`.

Click into the :guilabel:`Delivery Method` field to select an option from the drop-down menu.

Point Of Sale section
~~~~~~~~~~~~~~~~~~~~~

Under the :guilabel:`Point Of Sale` heading, enter a :guilabel:`Barcode` that can be used to
identify the contact. Use the :guilabel:`Loyalty Points` field to track points the user won as part
of a *Loyalty Program*.

Purchase section
~~~~~~~~~~~~~~~~

Specify :guilabel:`Payment Terms`, :guilabel:`1099 Box` information, and a preferred
:guilabel:`Payment Method` here. A :guilabel:`Receipt Reminder` can be set here, as well.

Misc section
~~~~~~~~~~~~

Under the :guilabel:`Misc.` heading, use :guilabel:`Reference` field to add any additional
information for this contact. If this contact should only be accessible for one company in a
multi-company database, select it from the :guilabel:`Company` field drop-down list. Use the
:guilabel:`Website` drop-down menu to restrict the publishing of this contact to one website (if
working on a database with multiple websites). Select one or more :guilabel:`Website Tags` to assist
in filtering published customers on the `/customers` website page. Select an :guilabel:`Industry`
for this contact from the drop-down menu. Use the :guilabel:`SLA Policies` field to assign a
*Helpdesk* SLA policy to this contact.
```

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:187]: Suggestion for specificity 
```suggestion
are being offered to this specific contact. It should be noted that this tab only appears when the
*Members* applications is installed.
```

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:240]: This information seems out of place in the seealso, I'd suggest putting it in a tip admonition, or removing it entirely 
```suggestion
```

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:14]: Adding this heading for skim-ability 
```suggestion

Contact type
------------

```

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:79]: I think this is a better image to use in place of `contacts/contact_form.png` because it has the contact's details filled out, and addresses shown. 

I'd like to see this image used (retaken without the markup on the Add button) in place of `contacts/contact_form.png` :).

Then, I don't think it is necessary to use an image here in the Contacts & Addresses tab, just to show the Add button.
```suggestion
```

File: content/applications/essentials/contacts.rst
Comment (samueljlieber) [inline content/applications/essentials/contacts.rst:97]: This image should come before the next instruction
```suggestion

.. image:: contacts/create-contact-window.png
   :align: center
   :alt: Create a new contact/address on a contact form.

Once an option is selected, enter the corresponding contact information that should be used for the
specified address type.
```

File: content/applications/essentials/contacts/contact_form.png
Comment (samueljlieber) [inline content/applications/essentials/contacts/contact_form.png:1]: Suggesting to remove this image, in replace of retaking `contacts/contact-form-add-address.png` without the markup on Add button

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @jero-odoo (and @tiku-odoo 😄), thank you for your work on this new essentials doc! This covers adding contacts nicely. I am requesting changes as I feel there are some restructuring needed in the content, please see my suggestions below! 

Tag me for another quick look after these have been addressed, thank you for your continued effort on this PR– its almost there!

**Comment by jero-odoo** [comment]
Hey @samueljlieber ready for another look. Thank you!!

**Review by samueljlieber — APPROVED** [review]
LGTM @jero-odoo, thank you!
.....
@robodoo r+

**Pull request closed without merging**
